docs/pages/architecture.md

```diff
@@ -24,6 +24,9 @@ Performance test reports area anonymously sent to Meshery Cloud for statistical
 - Meshery Consul Adapter: 10002/tcp
 - Meshery Octarine Adapter: 10003/tcp
 - Meshery NSM Adapter: 10004/tcp
+- Meshery App-Mesh Adapter:10005/tcp
```

Comment:
@ashiscs this is great! You can remove the hyphen ( - ) in the "App Mesh" name. Other than this, we're good to go.